quickstart/prerequisites.md

```diff
@@ -5,37 +5,208 @@
 All worker nodes must satisfy the following requirements:
 
 * **x86-64** CPU cores with SSE4.2 instruction support:
-  * Intel Nehalem processor \(march=nehalem\) and newer
+  * First generation Intel Core i5 and Core i7 (Nehalem microarchitecture) or newer
```

Comment:
CPU family references are being removed altogether by changes on the version 1.0 branch which should be incorporated here.